_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -64,9 +64,12 @@ default cmake.install_rpath         ${prefix}/lib
 #
 # cmake.generator Ninja
 # or
-# cmake.generator Unix Makefiles    # no quotes!
+# cmake.generator "Unix Makefiles"
+# or even
+# cmake.generator "Eclipse CDT4 - Ninja"
+# if maintaining the port means editing it using an IDE of choice.
 #
-default cmake.generator             {"Unix Makefiles"}
+default cmake.generator             "Unix Makefiles"
```

Comment:
That should stay `{"Unix Makefiles"}`, because the default values for the `default` statement should always have an extra level of quoting. Note that in this case, this isn't a list, but an extra level of Tcl escaping (technically there's no difference, of course).